README.md

```diff
@@ -10,6 +10,8 @@ See ["Caching dependencies to speed up workflows"](https://help.github.com/githu
 
 ## What's New
 
+* Updated the minimum runner version support from node 12 -> node 16.
+* Increased the cache size limit to 10 GB.
 * Added support for multiple paths, [glob patterns](https://github.com/actions/toolkit/tree/main/packages/glob), and single file caches.
```

Comment:
Should we remove the other items? They've been here for too long.

Reply:
True, that sounds reasonable. @bishal-pdMSFT, do you have any thoughts?

Reply:
My suggestion would be to group them by `v3` and `v2`.

Reply:
My suggestion would be to keep both v2 and v3 here for now. And add a CHANGELOG which will start summarizing the changes from 3.0.0 version onwards. Anytime we publish a new version in future, we should keep updating the changelog.